src/bundle/Resources/translations/messages.en.xliff

```diff
@@ -62,10 +62,15 @@
         <note>key: authentication.username</note>
       </trans-unit>
       <trans-unit id="b1b04388859d233f757e4f6376af3fc317d4baa6" resname="base.footer_copyright">
-        <source>eZ Platform ©1999-%current_year% %ez_link_open%eZ Systems AS%ez_link_close% and others.</source>
-        <target state="new">eZ Platform ©1999-%current_year% %ez_link_open%eZ Systems AS%ez_link_close% and others.</target>
+        <source>eZ Platform ©1999-%current_year% %ez_link_open%Ibexa AS%ez_link_close% and others.</source>
+        <target state="new">eZ Platform ©1999-%current_year% %ez_link_open%Ibexa AS%ez_link_close% and others.</target>
```

Comment:
```suggestion
        <target state="new">eZ Platform ©1999-%current_year% %ez_link_open%Ibexa%ez_link_close% and others.</target>
```

Reply:
Here I would say it applies the same approach as mentioned above (regarding "Ibexa AS" term) Ping @SylvainGuittard

Reply:
Sorry for the late reply on this. Next time do not hesitate to ping me on Slack.
It's fine to go with `Ibexa AS`. 👍